ftw/simplelayout/browser/client/web/js/app/simplelayout/Block.js

```diff
@@ -49,6 +49,13 @@ define([
       return this;
     };
 
+    this.markLinks = function() {
+      var url = window.location.protocol + "//" + window.location.host;
+      // all http links (without the link-plain class), not within this site
+      $("a[href^=\"http\"]:not(.link-plain):not([href^=\"" + url + "\"])", this.element).attr("target", "_blank");
```

Comment:
Uh, but it does respect the property in the plone integration: https://github.com/4teamwork/ftw.simplelayout/pull/388/files#diff-b9536bb9b91bb6a804502b377ca00adcR206